pillar/edx/ansible_vars.sls

```diff
@@ -73,23 +73,17 @@
         business_unit=business_unit,
         env=environment,
         purpose=purpose)) %}
-{% set remote_gradebook = salt.vault.read(
-    'secret-{business_unit}/{env}/remote_gradebook'.format(
-        business_unit=business_unit, env=environment)) %}
 {# END VAULT DATA LOOKUPS #}
 
 {# Begin Duplicated Variables #}
 {# multivariate #}
 {% set CMS_DOMAIN = purpose_data.domains.cms %}
-{% set EDXAPP_CMS_ISSUER = "https://{}/oauth2".format(CMS_DOMAIN) %}
+
 {% set COMMENTS_SERVICE_KEY = salt.vault.read('secret-residential/global/forum-api-key').data.value %} # TODO: randomly generate? (tmacey 2017/03/16)
-{% set DEFAULT_FEEDBACK_EMAIL = 'mitx-support@mit.edu' %}
-{% set DEFAULT_FROM_EMAIL = 'mitx-support@mit.edu' %}
 {# multivariate, needs to be different for Professional Education, sandbox, etc #}
 {% set GIT_REPO_DIR = edx.edxapp_git_repo_dir %}
 {# multivariate #}
 {% set LMS_DOMAIN = purpose_data.domains.lms %}
-{% set EDXAPP_LMS_ISSUER = "https://{}/oauth2".format(LMS_DOMAIN) %}
```

Comment:
This should stay in the base pillar config since it is actually more useful for sandboxes than residential deployments.